spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/EntityGraph.java

```diff
@@ -95,7 +94,7 @@ public enum EntityGraphType {
 
 		private final String key;
 
-		private EntityGraphType(String value) {
+		EntityGraphType(String value) {
 			this.key = value;
```

Comment:
In general, don't alter visibility settings without first consulting with the team. We have a strict policy to avoid breaking changes and generally discuss changes in visibility before carrying them out.

Reply:
Actually, this modifier has been redundant here because the enum is declared inside an annotation definition. So, this is not a breaking change.